RangeBarSample/src/main/java/com/appyvet/rangebarsample/RecyclerViewActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.appyvet.rangebarsample;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+
+public class RecyclerViewActivity extends Activity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Removes title bar and sets content view
+        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
```

Comment:
Please remove this and ensure that toolbar/actionbar is displayed with up navigation

Reply:
Done